OpenRA.Game/Orders/UnitOrderGenerator.cs

```diff
@@ -140,7 +140,12 @@ static UnitOrderResult OrderForUnit(Actor self, Target target, List<Actor> actor
 			if (mi.Button != Game.Settings.Game.MouseButtonPreference.Action)
 				return null;
 
-			if (self.Owner != self.World.LocalPlayer)
+			// TODO: What is the end goal of the spectating check?
```

Comment:
If not tested spectators could control any unit

Reply:
Makes sense, feel free to remove the TODO comment.